src/Bundle/Cli/DebugMailerCommand.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\MailerBundle\Cli;
+
+use Sylius\Bundle\MailerBundle\Cli\Dumper\DumperInterface;
+use Sylius\Bundle\MailerBundle\Cli\Dumper\EmailDetailDumperInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[AsCommand(name: 'sylius:debug:mailer', description: 'Shows configured emails and sender data')]
+final class DebugMailerCommand extends Command
+{
+    public function __construct(
+        /** @var DumperInterface[] $dumpers */
+        private readonly iterable $dumpers,
+        private readonly EmailDetailDumperInterface $emailDetailDumper,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this->addArgument('email', InputArgument::OPTIONAL, 'Email to be shown');
```

Comment:
I assume this is reference to `Model\Email` and not email address, then current naming strategy is somewhat confusing.

```suggestion
        $this->addArgument('emailCode', InputArgument::OPTIONAL, 'Email message to debug');
```

![image](https://github.com/user-attachments/assets/dbaab617-7d46-4649-8337-c54970d8fd9c)